scene_synchronizer.cpp

```diff
@@ -2006,7 +2006,9 @@ void ClientSynchronizer::process() {
 
 #ifdef DEBUG_ENABLED
 	if (unlikely(Engine::get_singleton()->get_frames_per_second() < physics_ticks_per_second)) {
-		WARN_PRINT("Current FPS is " + itos(Engine::get_singleton()->get_frames_per_second()) + ", but the minimum required FPS is " + itos(physics_ticks_per_second) + ", the client is unable to generate enough inputs for the server.");
+		if (!ProjectSettings::get_singleton()->get_setting("NetworkSynchronizer/show_only_errors")) {
+			WARN_PRINT("Current FPS is " + itos(Engine::get_singleton()->get_frames_per_second()) + ", but the minimum required FPS is " + itos(physics_ticks_per_second) + ", the client is unable to generate enough inputs for the server.");
```

Comment:
Can you please just change this line to `NET_DEBUG_PRINT`? So we don't need this extra `IF`. I think I forgot to use the net version here.